cookbook/templates/default/singularity.yaml.erb

```diff
@@ -0,0 +1,48 @@
+# singularity-related config:
+server:
+  type: simple
+  applicationContextPath: /
+  connector:
+    type: http
+    port: 7099
+  requestLog:
+    appenders:
+      - type: file
+        currentLogFilename: ../logs/access.log
+        archivedLogFilenamePattern: ../logs/access-%d.log.gz
+
+database:
+  driverClass: com.mysql.jdbc.Driver
+  user: <%= node[:singularity][:database][:username] %>
+  password: <%= node[:singularity][:database][:password] %>
+  url: jdbc:mysql://localhost:3306/<%= node[:singularity][:database][:db_name] %>
+
+mesos:
+  master: <%= node[:mesos][:master][:zk] %>
+  defaultCpus: 1
+  defaultMemory: 128
+  frameworkName: Singularity
+  frameworkId: Singularity
+  frameworkFailoverTimeout: 1000000
+
+zookeeper:
+  quorum: <%= node[:singularity][:zk_hosts].join(',') %>
+  zkNamespace: singularity
```

Comment:
this should be customizable, just in case one wants to run multiple singularity clusters on the same ZK cluster